src/protocol/protocol.h

```diff
@@ -1,6 +1,9 @@
 #ifndef PROTODEF
 
 #else
+#ifdef PROTO_HAS_RADIOLINK_CC2530
+PROTODEF(PROTOCOL_RADIOLINK,   RADIOLINK_CC2530, AETRG, RADIOLINK_Cmds, "Radiolink")
```

Comment:
You need keep RADIOLINK_CC2530 as a module name or use TX_MODULE_LAST which says it is available always. I prefer introduce a new module.